portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,88 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css">
     <link rel="stylesheet" href="style.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+    <script src="node_modules/jquery/dist/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <section class="hero is-light is-large">
+        <div class="hero-head">
+    <nav class="navbar">
```

Comment:
Starting here, the indentation is off. I believe this should be indented over and I think that might fix the indentation issues.

Reply:
i missed a </div> at the end of the file, just fixed it